host/tests/L3_TestCases/dsDisplay/dsDisplayHelperClass.py

```diff
@@ -63,13 +63,15 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
         #open Hal Session
         self.hal_session = self.dut.getConsoleSession("ssh_hal_test")
 
+        deviceTestSetup = self.cpe.get("test")
+
          # Create user response Class
         self.testUserResponse = utUserResponse()
 
         # Get path to device profile file
-        self.deviceProfile = os.path.join(dir_path, self.cpe.get("test").get("profile"))
+        self.moduleConfigProfileFile = os.path.join(dir_path, deviceTestSetup.get("profile"))
 
-        self.deviceDownloadPath = self.cpe.get("target_directory")
+        self.targetWorkspace = self.cpe.get("target_directory")
```

Comment:
do we need to have the target work space as well with the module name attached to it?  Probably we should follow a similar structure like in dsAudio.

Reply:
updated